File: task.js
// Task 2: Check a number is prime
function checkPrime(n) {
    if (n % 2 != 0 ) {
        console.log('is prime')
    } else {
        console.log('not prime')
    }
}

checkPrime(5)

// Task 4: 
function convertString(str) {
    console.log('Old String: ' + str)

    var newString = ''
    for (var i = str.length - 1; i >= 0; i--) {
        newString += str[i]
        
    }
    console.log('New String: ' + newString)
}

convertString('dong ngo')

// Task 5:
function findMax(arr) {
    let max = arr[0]
    for (var i = 0; i < arr.length; i++) {
        if (max < arr[i]) {
            max = arr[i]

            console.log(max)
        }
    }
}

findMax([3, 5, 2, 7, 9])

File: python/task.py
# Task 1:
def check():
    arr = []
    n = int(input('Enter your number of array: '))
    for i in range(0, n):
        item = int(input('Enter item in array: '))

        arr.append(item)

    k = int(input('Enter your number k: '))
    isInclude = False
    for ele in range(0, len(arr)):
        if arr[ele] == k: 
            isInclude= True
        
    if isInclude:
        print('Có tồn tại phần tử k trong mảng')
    else: 
        print('Không có tồn tại phần tử k trong mảng')

check()

# Task 2:
def checkPrime():
    n = int(input('Enter your prime number: '))
    if n % 2 != 0:
        print('is Prime')
    else:
        print('not a prime')

checkPrime()